models/Sketch/Sketch.test.js

```diff
@@ -35,6 +35,14 @@ describe('Sketch', () => {
       });
     });
 
+    it('should work when file is Buffer', () => {
+      expect(true).toBeTruthy();
+      const fileBuffer = fs.readFileSync(`${process.cwd()}/__tests__/__sketch-files/test.sketch`);
+      return Sketch.fromFile(fileBuffer).then(sketch => {
+        expect(sketch).toBeDefined();
```

Comment:
It's needed to manage the asynchrony. Use `done` callback or `async/await`.

Reply:
good call. Updated.